### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/dependencies/index.md`

### PR DESCRIPTION
**Pull request by KaniKim**
- Add Korean translation for docs/ko/docs/tutorial/dependencies/index.md
- I referred to #5605, Thanks to Author

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit cc06e3d4937bbd56d4003b03dbcc0e7c7e892ecf at: https://2cd29979.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f9d48d8fdaee75ae6896c99af96d5a9dc3821988 at: https://66197e59.fastapitiangolo.pages.dev

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
+# 의존성
+
+**FastAPI**는 아주 강력하지만 직관적인 **<abbr title="컴포넌트, 자원, 제공자, 서비스, 인젝터블로 알려져 있습니다">의존성 주입</abbr>** 시스템을 가지고 있습니다.
+
+이는 사용하기 아주 쉽게 설계했으며, 어느 개발자나 다른 컴포넌트와 **FastAPI**를 쉽게 통합할 수 있도록 만들었습니다.
+
+## "의존성 주입"은 무엇입니까?
+
+**"의존성 주입"**은 프로그래밍에서 여러분의 코드(이 경우, 경로 동작 함수)가 작동하고 사용하는 데 필요로 하는 것, "의존성"을 선언할 수 있는 방법을 의미합니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:9]:
```suggestion
**"의존성 주입"**은 프로그래밍에서 여러분의 코드(이 경우, 경로 동작 함수)가 작동하고 사용하는 데 필요로 하는 것, 즉 "의존성"을 선언할 수 있는 방법을 의미합니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+그 후에, 시스템(이 경우 FastAPI)은 여러분의 코드가 요구하는 의존성을 제공하기 위해 필요한 필요한 모든 작업을 처리합니다.(의존성을 "주입"합니다)
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:11]:
```suggestion
그 후에, 시스템(이 경우 FastAPI)은 여러분의 코드가 요구하는 의존성을 제공하기 위해 필요한 모든 작업을 처리합니다.(의존성을 "주입"합니다)
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+
+아주 간단한 예제를 봅시다. 너무 간단할 것이기에 지금 당장은 유용하지 않을 수 있습니다.
+
+하지만 이를 통해 **의존성 주입** 시스템이 어떻게 작동하는지에 중점을 둘 것입니다.
+
+### 의존성 혹은 "디펜더블" 만들기
+
+의존성에 집중해 봅시다.
+
+그저 단순한 *경로 작동 함수*가 가질 수 있는 모두 같은 매개변수를 가질 수 있는 함수입니다:
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:32]:
```suggestion
 *경로 작동 함수*가 가질 수 있는 모든 매개변수를 갖는 단순한 함수입니다:
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+그리고, 이 함수는 여러분의 모든 *경로 작동 함수*가 가지고 있는 것과 같은 형태와 구조를 가지고 있습니다.
+
+여러분은 이를 "데코레이터"가 없는 (`@app.get("/some-path")`가 없는) *경로 작동 함수*라고 생각할 수 있습니다.
+
+그리고 여러분이 원하는 무엇이든 반환할 수 있습니다.
+
+이 경우, 이 의존성은 다음과 같은 경우를 기대합니다:
+
+* 부가적인 쿼리 매개변수 `q`, `str`을 자료형으로 가집니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:82]:
```suggestion
* 선택적인 쿼리 매개변수 `q`, `str`을 자료형으로 가집니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+* 부가적인 쿼리 매개변수 `skip`, `int`를 자료형으로 가지며 기본 값은 `0`입니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:83]:
```suggestion
* 선택적인 쿼리 매개변수 `skip`, `int`를 자료형으로 가지며 기본 값은 `0`입니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+* 부가적인 쿼리 매개변수 `limit`,`int`를 자료형으로 가지며 기본 값은 `100`입니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:84]:
```suggestion
* 선택적인 쿼리 매개변수 `limit`,`int`를 자료형으로 가지며 기본 값은 `100`입니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+    !!! 팁
+        가능하다면 `Annotated`가 달린 버전을 권장합니다.
+
+    ```Python hl_lines="3"
+    {!> ../../../docs_src/dependencies/tutorial001.py!}
+    ```
+
+### "의존자"에 의존성 명시하기
+
+*경로 작동 함수*의 매개변수에서 `Body`, `Query` 등을 사용하는 방식과 똑같이 새로운 매개변수와 같이 `Depends`를 사용합니다:
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:135]:
```suggestion
*경로 작동 함수*가 매개변수로 `Body`, `Query` 등을 사용하는 방식과 같이 새로운 매개변수로 `Depends`를 사용합니다:
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+    {!> ../../../docs_src/dependencies/tutorial001.py!}
+    ```
+
+비록 `Body`, `Query` 등을 사용하는 것과 같은 방식으로 여러분의 함수의 매개변수에 있는 `Depends`를 사용하지만, `Depends`는 약간 다르게 작동합니다.
+
+`Depends`에 단일 매개변수만 전달했습니다.
+
+이 매개변수는 함수같은 것이어야 합니다.
+
+여러분은 직접 **호출하지 않았습니다** (끝에 괄호를 치지 않았습니다), 단지 `Depends()`에 매개변수처럼 넘겨 줬을 뿐입니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:179]:
```suggestion
여러분은 직접 **호출하지 않았습니다** (끝에 괄호를 치지 않았습니다), 단지 `Depends()`에 매개변수로 넘겨 줬을 뿐입니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+
+그리고 그 함수는 매개변수를 *경로 작동 함수*가 작동하는 것과 같은 방식으로 매개변수를 받습니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:181]:
```suggestion
그리고 그 함수는 *경로 작동 함수*가 작동하는 것과 같은 방식으로 매개변수를 받습니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+
+!!! 팁
+    여러분은 다음 장에서 함수를 제외하고서, "다른 것"이 어떻게 의존성으로 사용되는지 알게 될 것입니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:184]:
```suggestion
    여러분은 다음 장에서 함수를 제외하고서, "다른 것들"이 어떻게 의존성으로 사용되는지 알게 될 것입니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+그리고 일반적인 `def` *경로 작동 함수* 안에 `async def`로 의존성을 선언할 수 있으며, `async def` *경로 작동 함수* 안에 `def`로 의존성을 선언하는 등의 방법이 있습니다.
+
+아무 문제 없습니다. **FastAPI**는 무엇을 할지 알고 있습니다.
+
+!!! 참고
+    잘 모르시겠다면, [Async: *"In a hurry?"*](../../async.md){.internal-link target=_blank} 문서에서 `async`와 `await`에 대해 확인할 수 있습니다.
+
+## OpenAPI와 통합
+
+모든 요청 선언, 검증과 의존성(및 하위 의존성)에 대한 요구 사항은 동일한 OpenAPI스키마에 동합됩니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:264]:
```suggestion
모든 요청 선언, 검증과 의존성(및 하위 의존성)에 대한 요구 사항은 동일한 OpenAPI 스키마에 통합됩니다.
```

File: docs/ko/docs/tutorial/dependencies/index.md [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+
+따라서 대화형 문서에 이러한 의존성에 대한 모든 정보 역시 포함하고 있습니다:
+
+<img src="/img/tutorial/dependencies/image01.png">
+
+## 간단한 사용법
+
+이를 보면, *경로 작동 함수*는 *경로*와 *작동*이 매칭되면 언제든지 사용되도록 정의되었으며, **FastAPI**는 올바른 매개변수를 가진 함수의 호출하고 요청에서 데이터를 추출합니다.
```
Comment (KimJoonSeo) [inline docs/ko/docs/tutorial/dependencies/index.md:272]:
```suggestion
이를 보면, *경로 작동 함수*는 *경로*와 *작동*이 매칭되면 언제든지 사용되도록 정의되었으며, **FastAPI**는 올바른 매개변수를 가진 함수를 호출하고 해당 요청에서 데이터를 추출합니다.
```

**Review by KimJoonSeo — CHANGES_REQUESTED** [review]
I did some changes to make the sentences more clear.

**Comment by KaniKim** [comment]
> I did some changes to make the sentences more clear.

Thank you for your suggestions, I changed it, please check it if you can.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3d74851709129b39b19fb88c2dfc173cc2c91189 at: https://34b2bcbf.fastapitiangolo.pages.dev

**Review by KimJoonSeo — APPROVED** [review]

**Comment by alejsdev** [comment]
Thank you! @KaniKim 🎉 

And thanks for your review @KimJoonSeo 🧐🔍

**Pull request merged**